.kitchen.ec2.yml

```diff
@@ -8,7 +8,7 @@ end
 driver:
   name: ec2
   tags:
-    X-Project: chef-ci-acceptance
+    X-Project: chef-ingredient-acceptance
```

Comment:
Do you need the X-Dept and X-Contact tags as well?  
https://chefio.atlassian.net/wiki/display/ENG/AWS+Tagging+Standard